.github/workflows/CI.yml

```
@@ -39,6 +39,9 @@ jobs:
           - "3.8"
           - "3.7"
 
+    env:
+      TEST_ERROR_DIR: "$RUNNER_TEMP/test_errors"
```

```suggestion
      TEST_ERROR_DIR: "${{ runner.temp }}/test_errors"
```
Looks like something's wrong this variable.
https://github.com/pydot/pydot/actions/runs/7869461894/job/21468622150#step:9:508
Will this fix the problem?

---

Yeah, sorry, I'd meant to mark this as draft as it's not there yet. I can fix _that_ problem with `${{ env.RUNNER_TEMP }}` (or your suggestion, I'm sure), but the mismatched directory slashes on Windows still foul things.

---

(I added an `os.normpath()` to the `test_python.py` script, which fixes the directory slashes for the test run... but then they're still wrong when setting the `paths` for `actions/upload-artifact`. Frickin' Windows.)

---

Grrr. I'm beginning to suspect that `runner.temp` isn't defined yet, when setting envvars. I reduced it to just this, and still no go:

```yaml
env:
  TEST_ERROR_DIR: ${{ runner.temp }}
```

---

^ They weren't, so I switched to adding a step early in the job run that sets `TEST_ERROR_DIR` instead.